test/services/mapping/XmlMarshallingSpec.scala

```diff
@@ -63,14 +63,5 @@ class XmlMarshallingSpec extends WordSpec with Matchers with MockitoSugar {
 //    }
 //  }
```

Comment:
Can you remove above comments?

Reply:
I left the old test code there for whoever writes the new test

Reply:
I've updated the test now to test against the new model (and therefore got rid off all commented out lines)